### PR TITLE
PQ capabilites_ini_changes

### PR DESCRIPTION
**Pull request by ashishkumarrai1998**
1.2.0

### CODE REVIEW

**Review by ramkumarpattabiraman — DISMISSED** [review]
Changes approved.

File: config/pq_capablities.ini [hunk 1]
```diff
@@ -259,25 +259,29 @@ platformsupport = true
 
 [DimmingMode]
 range  = local,fixed
+index  = 1,0
```
Comment (upatel-sky) [inline config/pq_capablities.ini:262]: index is very generic word and can not relate what it relates to without checking code. Should we provide some documentation change for this?

**Review by upatel-sky — CHANGES_REQUESTED** [review]
index is very generic word and can not relate what it relates to without checking code. Should we provide some documentation change for this?

File: config/pq_capablities.ini (hunk 1 above)
Comment (utkarshece14) [reply config/pq_capablities.ini:262]: Chages approved

**Comment by utkarshece14** [comment]
Approved

**Review by utkarshece14 — APPROVED** [review]
Approved

**Pull request merged**